recipe-server/normandy/base/tests/test_urls.py

```diff
@@ -0,0 +1,7 @@
+def test_slash_redirects_have_cache_control_headers(client, settings):
```

Comment:
I wonder: Should we test the middleware directly? Asserting this for a single endpoint only feels a little fragile to future changes.